cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/CgmesReports.java

```diff
@@ -23,58 +24,58 @@ private CgmesReports() {
     // INFO
     public static ReportNode applyingPreprocessorsReport(ReportNode reportNode) {
         return reportNode.newReportNode()
-                .withMessageTemplate("applyingPreprocessors", "Applying preprocessors.")
+                .withLocaleMessageTemplate("core-cgmes-conversion-applyingPreprocessors", ReportBundleBaseName.BUNDLE_BASE_NAME)
```

Comment:
it's a bit strange to have snake-case mixed with camelCase :snake::camel: 
Could you use the dot character instead of the dash?